projects/packages/jetpack-mu-wpcom/src/features/wpcom-comments/wpcom-comment-like.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const displayNotice = message => {
+	const headerEnd = document.querySelector( '.wp-header-end' );
+	if ( ! headerEnd ) {
+		return;
+	}
+
+	// Create notice container.
+	const notice = document.createElement( 'div' );
```

Comment:
Should we attach something like `role="alert"` so that screen readers will announce the appearing notice? And is it enough to add that attribute? (There is also the manual `wp.a11.speak`, but I'm not sure that's needed.)

Reply:
Attached `role="alert"` to the notice and announcing the notice now.